### DISCUSSION TITLE
FlySky FS-GR3E receiver binding to MPM

### DISCUSSION THREAD

**Discussion by aseitz79** [pascallanger/DIY-Multiprotocol-TX-Module#933] [Q&A] [answered]
Hi there,

I am trying to bind my Radiomaster MT12 with MPM to a FlySky FS-GR3E receiver. I just can't get it to bind. :-(

![image](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/assets/52075616/f431d5e6-f481-44b3-a0e5-4c24698b4a7d)

So far I read that I need to set "FlSky2A" as Type in the MPM, but what do I need to set as subtype for this receiver? I tried everything without success. Holding down the bind button on the RX and selecting Bnd in the TX. Nothing worked.

Thanks for your help and kind regards,

Alex

**Comment by pascallanger** [comment, marked as answer]
The protocol for this RX is flysky without the 2A.

**Reply by aseitz79** [reply]
Hi Pascal,

thanks for the fast answer. It is working now. 🤗

Kind regards,

Alex

**Comment by onki69** [comment]
Is it possible that something has been changed in the FlySky protocol?
I was using the 3 channel RX with one of my models and a X20 (IRX4 Lite MPM module) and I can't connect neither bind anymore.
I did some firmware updates in the past as the model was not in use for at least one year.
The used protocol is "FlySky" and sub type "default". LED is blinking on the RX with binding plug installed.
I use the LBT firmware package. Could this be the problem?
Is there any document or info what protocols are in air, LBT and Surface package?

**Reply by pascallanger** [reply]
Just use air...

**Comment by onki69** [comment]
Thanks - now it works.

But where is an overview what version supports what protocol.
Maybe it is useful to add a column in the list of protocols and mention the version where the protocol is available.
Something like a A for air; S for surface ald L for LBT.

**Comment by pascallanger** [comment]
Just need time to do so... Feel free to submit a pull request if you want to contribute.
